src/rust/install.sh

```diff
@@ -202,6 +205,13 @@ fi
 echo "Installing common Rust dependencies..."
 rustup component add rls rust-analysis rust-src rustfmt clippy 2>&1
 
+if [ -n "${RUSTUP_TARGETS}" ]; then
+    for target in $(echo "${RUSTUP_TARGETS}" | sed -E 's/,/ /g'); do
+        echo "Installing additional Rust target $target"
+        rustup target add "$target" 2>&1
+    done
+fi
```

Comment:
```suggestion
if [ -n "${RUSTUP_TARGETS}" ]; then
    IFS=',' read -ra targets <<< "${RUSTUP_TARGETS}"
    for target in "${targets[@]}"; do
        echo "Installing additional Rust target $target"
        rustup target add "$target" 2>&1
    done
fi
```

Nit: Suggestion (untested)